tests/test_routes.py

```diff
@@ -84,4 +85,48 @@ def test_create(self):
         new_rec = resp.get_json()
         self.assertEqual(new_rec["pid"], rec.pid, "pid does not match")
         self.assertEqual(new_rec["recommended_pid"], rec.recommended_pid, "recommended_pid does not match")
-        self.assertEqual(new_rec["type"], rec.type, "type does not match")
\ No newline at end of file
+        self.assertEqual(new_rec["type"], rec.type, "type does not match")
+
+    def test_get(self):
+        """It should Get a Recommendation that is found"""
+        # Create a test case Recommendation
+        rec = make_recommendation(100, 200)
+        resp = self.client.post(
+            BASE_URL, json=rec.serialize(), content_type="application/json"
+        )
+
+        # Check GET status is 200_OK
+        location = resp.headers.get("Location", None)
+        resp = self.client.get(location, content_type="application/json")
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+
+        # Check GET data is correct
+        data = resp.get_json()
+        self.assertEqual(data["pid"], rec.pid)
+
+    def test_get_not_found(self):
+        """It should not Read a Recommendation that is not found"""
+        # Get a Recommendation that does not exist
+        resp = self.client.get(f"{BASE_URL}/0")
+        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_delete(self):
+        """It should Delete a Recommendation if exists"""
+        # Create a new Recommendation
+        rec = make_recommendation(100, 200)
+        resp = self.client.post(
+            BASE_URL, json=rec.serialize(), content_type="application/json"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
+
+        # Make sure location header is set
+        location = resp.headers.get("Location", None)
+        self.assertIsNotNone(location)
+
+        # Check that the location header was correct by deleting it
+        resp = self.client.delete("/recommendations/200", content_type="application/json")
+        self.assertEqual(resp.status_code, status.HTTP_204_NO_CONTENT)
+
+        # Check GET status is 404_NOT_FOUND
+        resp = self.client.get("/recommendations/200", content_type="application/json")
```

Comment:
The base URL for this test file is not yet changed, you have to update this line to 
```
resp = self.client.get(f"{BASE_URL}/200", content_type="application/json")
```

Reply:
I changed that.